zerver/openapi/zulip.yaml

```diff
@@ -4815,38 +4815,56 @@ paths:
       description: |
         Marks all of the current user's unread messages as read.
 
-        **Changes**: Before Zulip 6.0 (feature level 153), this
-        request did a single atomic operation, which could time out
-        with 10,000s of unread messages to mark as read.
-
-        It now marks messages as read in batches, starting with the
-        newest messages, so that progress will be made even if the
-        request times out.
-
-        If the server's processing is interrupted by a timeout, it
-        will return an HTTP 200 success response with result
-        "partially_completed". A correct client should repeat the
-        request when handling such a response.
+        Because this endpoint marks messages as read in batches, it is possible
+        for the request to time out after only marking some messages as read.
+        When this happens, the `complete` boolean field in the response will
+        be `false`, and a `code` field will be included in the response as well.
+        Clients should repeat the request when handling such a response. If all
+        messages were marked as read, then the success response will return
+        `"complete": "true"`.
+
+        **Changes**: As of Zulip 6.0 (feature level 153) and prior to to Zulip
+        8.0 (feature level 210), if the server's processing was interrupted by
```

Comment:
Yeah I agree with that feedback.

Reply:
Makes sense. Updated so that each **Changes** note steps back through the server history. Note that it would be good to review these notes for each endpoint because the changes for deleting a topic were made in two feature levels, while the mark all as read changes were made in one feature level.